ddsp/ai-platform/README.md

```diff
@@ -0,0 +1,103 @@
+# DDSP Docker
+
+Docker image for training autoencoder on [Google Cloud AI Platform](https://cloud.google.com/ai-platform).
+
+### Before you begin
+Make sure that you have completed the following steps:
+* Set up your [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+* Create a [Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/creating-buckets)
+* Enable [AI Platform Training and Prediction, Container Registry, and Compute Engine APIs](https://pantheon.corp.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component,containerregistry.googleapis.com)
+* Install [Docker](https://docs.docker.com/engine/install/)
```

Comment:
Just for clarity, it might be good to specify that all this setup is happening on whatever dev environment you prefer (local machine or GCE instance). I was confused for a bit on whether this required setting up a GCE instance for special cloud api stuff or something.

Or is the intent that the user would be using a GCE instance so the Docker build process goes faster?

Reply:
No, actually building the Docker image happens locally on the user side, we haven't tried building it on GCE instance. I will clarify this.

Reply:
Done